angular/src/app/shared/searchfields-list/searchfields-list.service.ts

```diff
@@ -20,7 +20,7 @@ export class SearchfieldsListService {
   constructor(private http: HttpClient,
     private appConfig: AppConfig,
     private cfg: AppConfig) {
-      this.RMMAPIURL = cfg.get("locations.mdService", "/unconfigured");
+      this.RMMAPIURL = cfg.get("API.mdSearch", "/unconfigured");
```

Comment:
This should be "APIs.mdSearch".  See [environment.ts](https://github.com/usnistgov/oar-pdr/blob/integration/angular/src/environments/environment.ts).

Reply:
I'll probably fix this.

Reply:
Oh ya, I didn't test my last minute change... Go ahead fix it please.